core/src/main/java/greencity/controller/OrderController.java

```diff
@@ -29,6 +30,9 @@
 public class OrderController {
     private final UBSClientService ubsClientService;
 
+    @Value("${green.city.client}")
+    private String greenCityClient;
```

Comment:
preferred way is to use ConfigProperties class instead of injecting @ Value here. @VadymMakitra used that approach in one of his PRs.